keel-actuator/src/test/kotlin/com/netflix/spinnaker/keel/actuation/EnvironmentPromotionCheckerTests.kt

```diff
@@ -45,10 +44,7 @@ internal class EnvironmentPromotionCheckerTests : JUnit5Minutests {
       publisher
     )
 
-    val artifact = DeliveryArtifact(
-      name = "fnord",
-      type = DEB
-    )
+    val artifact = DebianArtifact(name = "fnord")
```

Comment:
I had to change every place we refer to `DeliveryArtifact`, so there are a lot of these types of changes. You can ignore all of them because there was no functionality change.